<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitDateMinFilter</class>
 <widget class="QWidget" name="CommitDateMinFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>CommitDate Minimum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="uiCommitDateMin">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
